packages/suite/src/types/wallet/coinjoin.ts

```diff
@@ -53,3 +53,7 @@ export interface CoinjoinDebugSettings {
     coinjoinAllowNoTor?: boolean;
     coinjoinServerEnvironment?: PartialRecord<NetworkSymbol, CoinjoinServerEnvironment>;
 }
+
+export interface CoinjoinConfig {
+    averageAnonymityGainPerRound: number;
```

Comment:
perhaps [this](https://github.com/trezor/trezor-suite/blob/5f8f7e84ed477301911e1d06abd56a17056a8016/packages/suite/src/reducers/suite/suiteReducer.ts#L110) could be moved here? Although there is some storage-related logic related to it as well

Reply:
Currently a user can hide the exchanges banner and it's stored in the suite reducer and persisted. I remember someone mentioning that it should not be in the suite reducer.

<img width="788" alt="image" src="https://user-images.githubusercontent.com/45338719/217314734-40b63ad8-08ec-4750-8cc4-e4ac7140e49c.png">

Reply:
I see, let's move it to coinjoin reducer, but not inside config object - that is meant to be used for config from message system. Not part of this PR, right?

Reply:
sure!